datafusion/core/tests/sql/expr.rs

```diff
@@ -886,18 +886,6 @@ async fn csv_query_nullif_divide_by_0() -> Result<()> {
     Ok(())
 }
 
-#[tokio::test]
```

Comment:
this was randomly in the file for testing exprs when it is actually an (user defined) aggregate query  😕